### DISCUSSION TITLE
DNS load balancing and certificates

### DISCUSSION THREAD

**Discussion by Batistleman** [zodern/meteor-up#1288] [Q&A]
**Goal:** if one datacenter goes down (or one of the servers), I want the other to take over without downtime

**What I currently have:**
* 2 A records: 
    * blah.domain.com -> 1.2.3.4 (in region A)
    * blah.domain.com -> 2.3.4.5 (in region B)
* mup setup with two servers:
    * one: 1.2.3.4
    * two: 2.3.4.5
* servers have the same configuration, same db connections etc.
* loadbalancing is enabled, so when I deploy, there is no downtime already
* letencrypt setup takes care of the certificates for blah.domain.com

**Problem:**
When I shutdown server "one" completely, the other doesn't take over. I get a certificate error because the letsencrypt certificates are probably only available on one of the servers? (the one with the main proxy?)

Any ideas on how I could solve that?

**Comment by matt-dale** [comment]
I believe that there needs to be another system in place to manage the certificates. I'm going to attempt using a dedicated DO load balancer as suggested in this post: https://forums.meteor.com/t/recipe-scalable-high-availability-setup-with-meteor-up-digital-ocean-load-balancer/45433/17

**Reply by matt-dale** [reply]
I've had success using a Digital Ocean load balancer managing the certificate and the renewals. The Load Balancer decrypts the traffic and forwards it to the meteor servers. 

In the mup.js config, my root URL starts with `http://`, and I'm not using SSL.
